documentation/VandV/VandV.tex

```diff
@@ -115,26 +117,35 @@ \subsection{Naming Conventions and Terminology}
 
 \section{Testing Philosophy}
 \subsection{Approach}
-The approach to testing will be to separate the subsystems as much as possible and test them accordingly. Every subsystem will be unit tested in JUnit or PyUnit if implemented in Java or Python respectively. The shooting mechanism will be tested manually to ensure it is functional properly. Stubbing a service or dependancy can be done to return consistent, reliable and predictable results. For instance, the SmartServe system may use a stub for the CV subsystem to return a predefined sequence of pass/fail flags.
+The approach to testing will be to separate the subsystems as much as possible and test them accordingly. Every subsystem will be unit tested in JUnit or PyUnit if implemented in Java or Python respectively. The shooting mechanism will be tested manually to ensure it is functional properly. Stubbing a service or dependancy can be done to return consistent, reliable and predictable results. For instance, the SmartServe system may use a stub for the CV subsystem to return a predefined sequence of pass/fail flags. \\
+When preforming the tests, every test will be run three times unless specified otherwise. The test must be satisfied for 2 or 3 of the attempts to be determined successful.
 \subsection{Schedule}
 \begin{table}[H]
 \centering
-\caption{Testing Schedule}
 \label{my-label}
 \begin{tabular}{|l|l|l|}
 \hline
-\multicolumn{3}{|c|}{\textbf{Testing Schedule}}  \\ \hline
 \textbf{Task} & \textbf{Date} & \textbf{Notes} \\ \hline
 Complete Test Cases & February 13, 2018 & N/A \\ \hline
 Run Tests (First) & February 16, 2018 & N/A \\ \hline
 Edit Test Cases & March 9, 2018 & N/A \\ \hline
 Run Tests (Second) & March 16, 2018 & N/A \\ \hline
 \end{tabular}
+\caption{Testing Schedule}
 \end{table}
 \subsection{Environment}
 The SmartServe system uses heavy computation power due to the CV and ML models, so the system which runs the test will have the following details. The system will be a 15-inch Macbook Pro (Late 2016) running macOS High Sierra (10.13). The Macbook Pro has an 2.6 GHz Intel Core i7 CPU and a Radeon Pro 450 2GB GPU. The location will be in Thode Makerspace where the CV will be adjusted as necessary for those lighting conditions.
 \subsection{Code Reviews}
 As per the Development Plan document, the team will be using branches on GitHub to control additions into the source code. The list of Pull Requests can be found here \href{https://github.com/ChristopherMcDonald/SoftwareTronCapstone/pulls?utf8=?&q=is\%3Apr}{here}. Although it is impossible to test whether it is better that they were made as opposed to all members contributing to the same branch, few commits have had to be made directly to the develop branch, and 0 to the master branch, which indicates the system is working as expected.
+\subsection{Setup Instructions}
```

Comment:
Love this. <3

Reply:
We should test on someone's laptop to be sure I covered everything, which will provide redundancy for the demoes anyway.